### DISCUSSION TITLE
Unable to add aqara sensors to a group

### DISCUSSION THREAD

**Discussion by dannyv1101** [Koenkk/zigbee2mqtt#22135] [Devices]
Hi,

I tried to add sensors from aqara to a group but this did not work. Lightbulbs and other actors I can add to a group but if I try to add a sensor nothing happens. I tried it with aqara motion, door and multisensors (temp, humidity and pressure).

Maybe there is something wrong in my config file?

**Comment by mrskycriper** [comment]
What are you trying to accomplish? Non controllable devices don't support groups most of the time. The group primary function is to sync the controls for all devices inside.
